docs/content/preview/manage/backup-restore/instant-db-cloning.md

```diff
@@ -0,0 +1,292 @@
+
+## Enable database cloning
+
+To enable database cloning in a cluster, you set the YB-Master flag `enable_db_clone` to true. Because cloning is in {{<badge/tp>}}, you must also add the `enable_db_clone` flag to the [allowed_preview_flags_csv](../../../reference/configuration/yb-master/#allowed-preview-flags-csv) list.
+
+For example, to set these flags when creating a cluster using yugabyted, use the `--master_flags` option of the [start](../../../reference/configuration/yugabyted/#start) command as follows:
+
+```sh
+./bin/yugabyted start advertise_address=127.0.0.1 \
+    --master_flags "allowed_preview_flags_csv={enable_db_clone},enable_db_clone=true" \
+    --tserver_flags "ysql_hba_conf_csv={host all all 0.0.0.0/0 trust,local all all trust}"
+```
+
+You can set the runtime flags while the yb-master process is running using the yb-ts-cli [set_flag](../../../admin/yb-ts-cli/#set-flag) command as follows:
+
+```sh
+./bin/yb-ts-cli --server-address=master_host:7100 set_flag allowed_preview_flags_csv enable_db_clone
+./bin/yb-ts-cli --server-address=127.0.0.1:7100 set_flag enable_db_clone true
+```
+
+## Clone databases
+
+### Prerequisites
+
+- [Create a snapshot schedule](../../../manage/backup-restore/point-in-time-recovery/#create-a-schedule) for the database you want to clone.
+
+    For example, creating a snapshot schedule with retention period of 7 days keeps the history of the updates for the entire 7 days. After that you can create a clone of the original database as of any point in time within the specified retention period.
+
+- You have to trust local YSQL connections (that use UNIX domain sockets) in the [host-based authentication](../../../secure/authentication/host-based-authentication/). You have to do this for all YB-TServers in the cluster. You can do this when starting the YB-TServer process by adding the authentication line `local all all trust` to the [ysql_hba_conf_csv](../../../reference/configuration/yb-tserver/#ysql-hba-conf-csv) flag.
+
+    For example, if you are using yugabyted you can run the following command:
+
+    ```sh
+   ./bin/yugabyted start --advertise_address=127.0.0.1 \
+       --master_flags "allowed_preview_flags_csv={enable_db_clone},enable_db_clone=true" \
+       --tserver_flags "ysql_hba_conf_csv={host all all 0.0.0.0/0 trust,local all all trust}"
+    ```
+
+{{<note title="Note">}}
+Do not override your default host-based authentication rules when trusting the local connection. You may need to add additional authentication lines to `ysql_hba_conf_csv` based on your specific configuration. For more information, see [host based authentication](../../../secure/authentication/host-based-authentication/).
```

Comment:
Will setting the flag override these settings? We need to be more clear here about when we would override the default rules